api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -182,7 +182,7 @@ def test_citation_no_repeated_periods(self):
     def test_citation_osf_provider(self):
         self.node.title = 'A Study of Coffee.'
         self.node.save()
-        self.published_preprint.provider.name = 'Open Science Framework'
+        self.published_preprint.provider.name = 'OSF Preprints'
```

Comment:
Does this need to be changed? Shouldn't the special casing above do this on serialization? It seems like the `assert_equal` call below is what should be modified